src/content/docs/en/recipes/bun.mdx

```diff
@@ -2,86 +2,11 @@
 title: Use Bun with Astro
 description: Learn how to use Bun with your Astro site.
 type: recipe
-i18nReady: true
+i18nReady: false
 ---
 
 [Bun](https://bun.sh/) is a runtime for JavaScript built for speed. See [Bun's documentation](https://bun.sh/docs) for more information.
 
-Using Bun with Astro still is experimental. Some integrations may not work as expected. If you have [any problems using Bun](https://github.com/oven-sh/bun/issues/new/choose), please [open an issue on GitHub](https://github.com/withastro/astro/issues/new/choose)!
-
-## Installing Bun
-
-Use the following command to install Bun:
-
-
-```bash
-curl -fsSL https://bun.sh/install | bash
-```
-
-## Create a new Astro project with Bun
-
-Create a new Astro project with Bun using the following `create astro` command:
-
-```bash
-bunx create-astro@latest my-astro-project-using-bun
-```
-
-:::tip[Tip]
-You can also [create a new Astro project from any existing Astro GitHub repository](/en/install/auto/#starter-templates) using the `--template` flag:
-```bash
-bunx create-astro@latest my-astro-project-using-bun --template eliancodes/brutal
-```
-:::
-
-## Install your dependencies
-
-If you're starting a new project using `bunx create-astro`, the CLI will automatically use Bun to install dependencies and you can skip this step. 
-
-Otherwise, you'll need to install your dependencies with Bun:
-
-```bash
-bun install
-```
-
-## Running the development server
-
-To run the development server, use the following command:
-
-```bash
-bun run dev
-```
-
-:::note[Note]
-Use the full command `bun run dev` to start your Astro development server. The command `bun dev` will start Bun's own internal dev server.
-:::
-
-## Using Astro integrations
-
-You can also use any of the official Astro integrations with Bun and the `astro add` command:
-
-```bash
-bunx astro add react
-```
-
-This will work exactly the same as if you were using NPM, but with the added benefit of using Bun's blazing fast runtime.
-
-:::note[Note]
-Some integrations might not work as expected. If you find any issues, please [open an issue on GitHub](https://github.com/withastro/astro/issues/new/choose)!
-:::
-
-## Building your site with Bun
-
-To build and serve your site, Bun has familiar commands:
-
-```bash
-bun run build
-```
-
-Bun's build command will output your site to the `dist/` directory.
-
-Then, you can serve your site using the `preview` command:
-
-```bash
-bun preview
-```
+Using Bun with Astro now officially supported but is still very new. Some integrations may not work as expected. Consult [Bun's official documentation for working with Astro](https://bun.sh/guides/ecosystem/astro) for details. Our community is busy at work preparing our own guide for using Bun in Astro! 
```

Comment:
```suggestion
Using Bun with Astro is now officially supported but is still very new. Some integrations may not work as expected. Consult [Bun's official documentation for working with Astro](https://bun.sh/guides/ecosystem/astro) for details. Our community is busy at work preparing our own guide for using Bun in Astro! 
```

Reply:
Awesome, I added a small suggestion.

Reply:
It looks like this already is resolved! 🙌

Reply:
Thank you @sasoria ! Yup, I caught it myself around the same time!  Nicely spotted!  💜